WINDOWS_SETUP.md

```diff
@@ -0,0 +1,46 @@
+# Windows Development and Testing 
+
+The Shield Studies Addon Template makes some assumptions about your environment that can be challenging to meet on Windows machines. So far the most promising approach uses the **Windows Subsystem for Linux (WSL)**. WSL is a young project with bugs and unexpected pitfalls; caveat emptor. 
+
+## Requirements
+
+* Windows Version 10.0.14986+ (find your version by typing `ver` at Windows command line)
+
+## Installing WSL
+
+1. [Follow Microsoft's official steps for installing WSL](https://answers.microsoft.com/en-us/insider/wiki/insider_wintp-insider_install/how-to-enable-the-windows-subsystem-for-linux/16e8f2e8-4a6a-4325-a89a-fd28c7841775?auth=1). These instructions are clear and detailed. _If you prefer, here is a TL;DR:_
+    1. Enable developer mode in Windows 10 in `Start > Settings > Update & security > For developers`.
+    2. Enable the optional Windows feature, "Windows Subsystem for Linux" using `optionalfeatures.exe`.
+    3. Restart.
+    4. Type `bash` at the Windows command line and wait for Ubuntu to install.
+
+2. Install a recent version of node.js:
+```
+curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+sudo apt-get install -y nodejs
+```
+3. Get the latest npm: `npm install npm@latest -g`
+
+4. Install git and zip (and any other linux command-line tools you like):
+```
+sudo apt install git
+sudo apt install zip
+```
+
+5. Now you can follow the standard instructions in the README at [https://github.com/mozilla/shield-studies-addon-template](https://github.com/mozilla/shield-studies-addon-template) for cloning the repo, installing npm modules and running npm scripts.
```

Comment:
Nit: Could you use a relative link here?
`... instructions in the [README](/README.md)`

Reply:
Actually I changed my mind, sorry. This is fine. I don't know that there is a relative URL for the repo home. The link above wouldn't have the desired effect.